pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -196,7 +196,7 @@ func (this *SnapshotManager) DeleteSnapshot(peID astrolabe.ProtectedEntityID) er
 	}
 
 	log.Infof("Step 2: Deleting the durable snapshot from s3")
-	err = this.DeleteProtectedEntitySnapshot(peID, true)
+	err = this.DeleteRemoteProtectedEntitySnapshot(peID)
```

Comment:
So this returns an error if we were unable to delete the durable snapshot.  Can we check for "not found here"  If we have a snapshot that has not uploaded to S3 and we go to remove the backup, it's possible that the snapshot exists locally but not in the durable repository